app/routes/surveys/components/Question.js

```diff
@@ -0,0 +1,237 @@
+// @flow
+
+import React, { Component } from 'react';
+import Option from './Option';
+import { TextInput, TextArea, CheckBox } from 'app/components/Form';
+import Dropdown from 'app/components/Dropdown';
+import styles from './surveys.css';
+import Icon from 'app/components/Icon';
+import { Link } from 'react-router';
+import { ConfirmModalWithParent } from 'app/components/Modal/ConfirmModal';
+
+type Props = {
+  updateQuestion: Object => void,
+  deleteQuestion: number => Promise<*>,
+  question: Object,
+  nr: number
```

Comment:
what's an `nr` (give it a better name :p)

Reply:
Yeah I wasn't 100% sure what it would end up as when I implemented it, I just needed some identifier for objects that didn't have an assigned id from the backend yet. I agree index is a better name, just wasn't sure it would end up being akin to an array index which might have made the name confusing. Probably still better than `nr` tho